server/server_nonlinux.go

```
@@ -0,0 +1,21 @@
+// +build !linux
+
+package server
+
+import "net"
+
+type server struct{}
+
+// NewServer constructs a new unstarted packetforward Server. The server can be started by
+// calling Serve(). On non-linux platforms, this server does nothing.
+func NewServer(opts *Opts) (Server, error) {
+	return &server{}, nil
```

Perhaps all of these functions should return an error (e.g. `errors.New("unsupported OS")`).  What do you think?

---

Good idea, will do.